x-pack/plugins/security_solution/public/cases/pages/configure_cases.tsx

```diff
@@ -32,14 +33,15 @@ const ConfigureCasesPageComponent: React.FC = () => {
     () => ({
       href: getCaseUrl(search),
       text: i18n.BACK_TO_ALL,
-      pageId: SecurityPageName.case,
+      pageId: SecurityPageName.case as SiemNavTabKey,
```

Comment:
nit - why do we need this cast?

Reply:
This issue came from `useFormatUrl`, that needs a `pageId` that is in the home_navigation (the main one), so it needs a `SiemNavTabKey` which is a subset of the `SecurityPageName` enum. 
The thing is now we have pages that are needed for the deepLinks but are not in the main navigation. 
We can do some type cleaning in a following PR.